README.md

```diff
@@ -7,6 +7,19 @@ hardware-specific and kernel-specific quirks.
 
 libratbag exports the devices over a DBus daemon called **ratbagd**, see the
 [ratbagd DBus Interface description](https://libratbag.github.io/).
+Users interact through a GUI like
+[Piper](https://github.com/libratbag/piper/). For developers, the
+`ratbagctl` tool is the prime tool for debugging.
+
+Installing libratbag from system packages
+-----------------------------------------
+
+libratbag is packaged for some distributions, you can use your system's
+package manager to install it:
+- Fedora: `dnf install libratbag-ratbagd`
+- Arch: `pacman -S libratbag`
+- Debian/Ubuntu: **do not use the system packages, they haven't been updated
+  in years**, install from source instead
```

Comment:
Don't we have someone maintaining a PPA for Ubuntu? (@muzena)

Reply:
there are [a few](https://launchpad.net/ubuntu/+ppas?name_filter=libratbag) but I couldn't quite tell if any of them are really maintained. Esp. given that 18.10 was released since and no-one seems to have bothered with the ratbag packages there.

Reply:
Note: I've now changed the commit to refer to [the wiki](https://github.com/libratbag/libratbag/wiki/Installation) where we can easily update this.